jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/TemplateSettings.kt

```diff
@@ -60,9 +62,14 @@ class TemplateSettings(val project: Project) {
                 project,
                 FileChooserDescriptorFactory.createSingleFileDescriptor(YAMLFileType.YML)
             ) {
-                setTemplateFile(it.canonicalPath)
+                templateFile.text = it.canonicalPath ?: ""
             }
         )
+        templateFile.textField.document.addDocumentListener(object : DocumentAdapter() {
+            override fun textChanged(e: DocumentEvent) {
+                updateFunctionModel(templateFile.text)
```

Comment:
Not that I could tell (on my very slow computer), but potentially if they have an extremely large template. I assume most users will copy paste or use the folder browser since it needs a full path